pkg/apis/upgrade/v1alpha1/upgradeconfig_types.go

```diff
@@ -12,6 +12,13 @@ type UpgradeConfigSpec struct {
 	// Specify the desired OpenShift release
 	Desired Update `json:"desired"`
 
+	// Specify the upgrade start time
+	UpgradeAt string `json:"upgradeAt"`
+
+	// +kubebuilder:default:=true
+	// Procceed whether to upgrade or not, this gives operator the ability to cancel the upgrade if upgrade hasn't started
```

Comment:
Very minor spelling/grammar fix
```suggestion
	// Proceed whether to upgrade or not, this gives the operator the ability to cancel the upgrade if the upgrade hasn't started
```